### PR TITLE
ST-490 Summary page for flags

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-490

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by spotstory — DISMISSED** [review]

**Review by muradalamv1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/model/CaseFlag.java [hunk 1]
```diff
@@ -64,4 +68,78 @@ public class CaseFlag {
...
+    )
+    private List<ListValue<CaseFlagDisplay>> partyLevelFlags;
+
+    @CCD(
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private List<ListValue<CaseFlagDisplay>> caseLevelFlags;
+
+    @SuppressWarnings("unchecked")
+    public List<ListValue<CaseFlagDisplay>> getPartyLevelFlags() {
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/model/CaseFlag.java:82]: Do we need SuppressWarnings("unchecked") here? Is it possible to code the generics so as to be able to remove this?

Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/model/CaseFlag.java:82]: Could not find a way to work without them

Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/model/CaseFlag.java:82]: I checked out the branch to take a look. The first 2 methods aren't used, so it doesn't really check whether the suppression is needed or not, and the 3rd suppression is apparently redundant, so can be removed?

**Review by sjamihmcts — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**